packages/args.ol

```diff
@@ -0,0 +1,115 @@
+from string-utils import StringUtils
+from console import Console
+
+
+type GetParsedArgsRequest {
+    args*: string 
+    argsMap*: void {
```

Comment:
Is there something more semantic we could call this? `argTypes`, `argsSpec`, `schema`, ...?